packages/lib/src/composables/tooltip.ts

```diff
@@ -63,10 +80,14 @@ export function useTooltipAnchors(
   );
 
   function updateTooltipAnchorAttributes(renderedData: InternalData) {
+    const numberOfLabels = labels?.value?.length ?? 0;
     const highestValues =
       chartType?.value && ANCHOR_CALCULATION_METHOD_MAP[chartType.value]
-        ? ANCHOR_CALCULATION_METHOD_MAP[chartType.value](renderedData)
-        : getHighestValues(renderedData);
+        ? ANCHOR_CALCULATION_METHOD_MAP[chartType.value](
+          renderedData,
+          numberOfLabels
+        )
+        : getHighestValues(renderedData, numberOfLabels);
```

Comment:
Don't need a constant if you're not reusing the value:

```suggestion
        ? ANCHOR_CALCULATION_METHOD_MAP[chartType.value](
          renderedData,
          labels?.value?.length ?? 0
        )
        : getHighestValues(renderedData, numberOfLabels);
```

Reply:
`numberOfLabels ` is used twice :)

one as the second parameter to `getHighestValues()` and the other one as second param to  `ANCHOR_CALCULATION_METHOD_MAP[chartType.value]()`

Reply:
ah right, then it's ok